app/models/conversation.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Conversation < ApplicationRecord
+  has_and_belongs_to_many :users
```

Comment:
we prefer user `has_many :through`
[RoR guides](https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#models)